app/assets/stylesheets/application.scss

```diff
@@ -130,15 +144,44 @@ h1 {
   text-align: center;
 }
 
+
+.searchbra {
+  margin-left: 0;
+}
 .button {
   display: block;
+  padding: 10px;
   margin: 0 auto;
+  color: #000000;
+}
+
+.button:hover {
+  background-color: #6FDF52;
+  color: #000000;
+}
+
+h5 {
+  text-align: center;
 }
 
 #created {
   color: white;
 }
 
-/*.footer ul {
-  float: right;
-}*/
+h1 a {
+  text-align: center;
+}
+
+h1 {
+  color: #FF0082;
+}
+
+h2 {
+  color: #FF0082;
```

Comment:
Color literals like `#FF0082` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Color `#FF0082` should be written as `#ff0082`